src/components/About/about.css

```diff
@@ -0,0 +1,106 @@
+.about-wrapper {
+  width: 340px;
+  margin: 0 auto;
+  border-radius: 20px;
```

Comment:
border-radius value should be variable as well.

Reply:
done